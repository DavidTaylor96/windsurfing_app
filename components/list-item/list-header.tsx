import * as React from 'react';
import { FC } from 'react';
import { StyleSheet } from 'react-native';
import { Text, View } from '../Themed';

interface IListHeader {}

export const ListHeader: FC<IListHeader> = (props) => {
  return (
    <View style={styles.header}>
      <Text weight="h1">{props.children}</Text>
      <View style={styles.underline} />
    </View>
  );
};

const styles = StyleSheet.create({
  header: {
    backgroundColor: 'white',
    marginHorizontal: 20,
    paddingVertical: 10,
  },

  underline: {
    height: 2,
    width: 40,
    backgroundColor: '#F401F2',
    marginTop: 3,
  },
});
