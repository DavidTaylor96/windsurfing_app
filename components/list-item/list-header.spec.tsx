import * as React from 'react';
import renderer from 'react-test-renderer';
import { ListHeader } from './list-header';

it(`List Headeru`, () => {

  const listHeader = renderer.create(
    <ListHeader>Header</ListHeader>
  );

  expect(listHeader).toMatchSnapshot();
});
