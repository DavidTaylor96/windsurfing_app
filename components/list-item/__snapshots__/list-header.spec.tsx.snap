// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Headeru 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "marginHorizontal": 20,
      "paddingVertical": 10,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "fontFamily": "sora-regular",
          "fontSize": 24,
        },
        Object {
          "color": "black",
        },
        undefined,
      ]
    }
    weight="h1"
  >
    Header
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "#F401F2",
        "height": 2,
        "marginTop": 3,
        "width": 40,
      }
    }
  />
</View>
`;
